contracts/SimpleNftMarketplace.sol

```diff
@@ -16,8 +16,8 @@ contract SimpleNftMarketplace is ListingManager, ValidateSignature {
     return _version;
   }
 
-  function createListing(address tokenContract, uint256 tokenId, uint256 salePrice) external returns (uint256 listingId) {
-    _createListing(tokenContract, tokenId, salePrice);
+  function createListing(address tokenContract, uint256 tokenId, uint256 salePrice, address seller) external returns (uint256 listingId) {
+    _createListing(tokenContract, tokenId, salePrice, seller);
```

Comment:
We don't need seller as a argument in the external function, we will just hardcode msg.sender when using _createListing (same for buyListing external function)

Reply:
i did it in the same time, commit "improve createListing issue#21"